samples/tfx-oss/README.md

```diff
@@ -39,9 +47,9 @@ Modify the pipeline configuration file at
 tfx/examples/chicago_taxi_pipeline/taxi_pipeline_kubeflow_large.py
 ```
 Configure 
-- GCS storage bucket name (replace "my-bucket")
-- GCP project ID (replace "my-gcp-project")
-- Make sure the path to the taxi_utils.py is correct
+- Set `_input_bucket` to the GCS directory where you've copied taxi_utils.py. I.e. gs://<my bucket>/<path>/
+- Set `_output_bucket` to the GCS directory where you've want the results to be written
+- Set GCP project ID (replace my-gcp-project). Note that it should be project ID (usually has numbers in the end), not project name.
```

Comment:
The numbers in the end bit isn't true, so I won't point that out. GCP project id is sufficient. Drop the it from 'Note...' ?

Reply:
Done.